File: 백준/Bronze/10203. Count Vowels/Count Vowels.c
#include <stdio.h>
#include <string.h>
int main(void) {
    int n, cnt;
    char st[100], vowel[5] = {'a', 'e', 'i', 'o', 'u'};
    scanf("%d", &n);
    for (int i = 0; i < n; i++) {
        cnt = 0;
        scanf("%s", st);
        for (int j = 0; j < strlen(st); j++) {
            for (int k = 0; k < 5; k++) {
                if (st[j] == vowel[k]) {
                    cnt += 1;
                    break;
                }
            }
        }
        printf("The number of vowels in %s is %d.\n", st, cnt);
    }
    return 0;
}

File: 백준/Bronze/10203. Count Vowels/Count Vowels.py
import sys
input=lambda:sys.stdin.readline().rstrip("\n")
for _ in range(int(input())):
    st,vowel,cnt=input(),['a','e','i','o','u'],0
    for c in st:
        if c in vowel:
            cnt+=1
    print(f"The number of vowels in {st} is {cnt}.")